### DISCUSSION TITLE
Blank Foundry Screen on Port 30000

### DISCUSSION THREAD

**Discussion by TheForlornKnight** [felddy/foundryvtt-docker#410] [Q&A]
I got it into my head to try running off my Synology DS918+ and having difficulties. I've been working off of [This Guide](https://foundryvtt.wiki/en/setup/hosting/Synology) which has got me most of the way. 
Right now I've got Foundry running and can get to it from :30000. The first time I did, a field asked for my license key, which I entered. Since then, the url is now :30000/auth but it's just a blank dark screen with the foundry logo. No field to enter or set my admin key or otherwise log in. I've tried creating an admin key in the environment but that hasn't changed anything. Don't know enough to know where to go from here.
Similar discussions #118  and #278 both seem to be with proxies but have port 30000 working, whereas I can't get 30000 to work
![Settings 01](https://user-images.githubusercontent.com/97814843/186040650-5d3b7169-f3ad-4c16-8464-e6c24bfca27a.jpg)
![Settings 02](https://user-images.githubusercontent.com/97814843/186040656-d0dc2607-0368-45f0-84fd-11618b0131e7.jpg)
![Log 01](https://user-images.githubusercontent.com/97814843/186041145-884f40cd-bbd1-47c6-b6ad-8afe63fe6767.jpg)
![Log 02](https://user-images.githubusercontent.com/97814843/186041153-83133b57-5d52-4808-a3a4-ae74ee1a5efe.jpg)
.

**Comment by mckee14** [comment]
You don't need the option "enable web portal via web station". Im not entirely sure what it does, but the felddy image creates a internal port 30000 on which foundry is listening.
You need to set the external port for the container which is then routed to the internal foundry port inside the container:
![image](https://user-images.githubusercontent.com/62849695/187760214-bf3c9c39-e34e-481a-af3a-31e013c3e847.png)

then, you try to log into foundry from inside your network via http://nas-ip:51234

when you can see the logo and nothing else, that usually is a problem with a websocket. I only had this when I started using a reverse proxy. Maybe in your case it's because you enabled the web portal on port 30000.
